src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.cs

```diff
@@ -197,8 +198,10 @@ public Bitmap GetThumbnailImpl(uint cx)
                         _browser.NavigateToString(SvgContents);
                     }
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
+                    Logger.LogError($"Failed running webView2Environment completed for {FilePath} : ", ex);
+                    thumbnailDone.Set();
```

Comment:
Should this be done in a finally statement vs the catch?

Reply:
There's a success path where we need to run `browser.NavigationCompleted` and that's the one that ends up setting the process.